docker-ci/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Parent image for zingolibs CI
+FROM parent-ci:latest
+
+# RUN git clone https://github.com/zingolabs/zingolib.git
```

Comment:
I get why you're doing this now, and I think it's the right way, but once the image is in CI, I think the version we want to test against will be available through the github CI interface.

Reply:
ok yes we are moving onto integrating to github now so will hook the parent image up to the CI workflow